gen_docker_compose.py

```diff
@@ -79,6 +79,11 @@ def generate_compose_file(
         for k, v in self.config['versions'].items():
             template_args[k] = v
 
+        short_git_hash_key = "SHORT_GIT_HASH"
+
+        if os.environ.get(short_git_hash_key) is not None:
+            template_args[short_git_hash_key] = os.environ[short_git_hash_key]
```

Comment:
You can simplify this by doing:

```
template_args['SHORT_GIT_HASH'] = os.getenv['SHORT_GIT_HASH, 'Unknown']
```

(Make sure variables match etc but the point is you can avoid the logic and just set a default in `os.getenv`)